client/OPUNetGameSelectWnd.h

```diff
@@ -3,7 +3,8 @@
 
 using namespace OP2Internal;
 
-const int MaxServerAddressLen = 128;
+const int MaxServerAddressLength = 128;
+const int MaxPlayerNameLength = 13;
```

Comment:
A mix of 13, 16, and 128 were used for player name length in different portions of OP2Internal and OPUNetFixClient.

I Went with 13 because it is used in HostGameParameters::gameCreateName, although didn't do maybe as exhaustive search as should have.

Reply:
Good call. The previously used constant was perhaps not the most appropriate for all uses of it.